test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -768,6 +768,36 @@ def forward(self, x):
                 check_quantization=True,
             )
 
+    @skipIfNoDynamoSupport
+    @skipIfNoONEDNN
+    @skipIfRocm
+    def test_qlinear_add(self):
+        r"""
+        This testcase will quantize a Linear->Add pattern.
```

Comment:
`Linear->Add` is a fusion pattern in our near further plan, can we change the pointwise op to sub or mul?

Reply:
Done, thanks.